content/docs/command-reference/remote/modify.md

```diff
@@ -294,6 +294,41 @@ $ dvc remote modify myremote endpointurl \
 For more information on configuring Azure Storage connection strings, visit
 [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string).
 
+**Advanced Azure Authentication**
+
+If you rely on AAD to authenticate against your azure storage account, 
```

Comment:
is it AD or AAD? just to make sure

Reply:
I think either "AD" or "Azure AD" would be most correct. In this case just "AD" since "Azure" is later in the same sentence.

Reply:
```suggestion
If you rely on
[Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis)
to authenticate against your Azure Storage account,
```